tests/cases/fourslash/getOutliningSpansInLargeFile.ts

```diff
@@ -0,0 +1,2457 @@
+// ////var c/*68*/3_i_2 = new c/*60q*/3(/*60*/"hello");
+// ////var c4/*69*/_i_1 = new c/*61q*/4(/*61*/10);
+// ////var c4_i_2 = new c/*62q*/4(/*62*/"hello");
+// ////var c/*70*/5_i_1 = new c/*63q*/5(/*63*/10);
+// ////var c5_i_2 = new c/*64q*/5(/*64*/"hello");
+// /////** This is multiOverload F1 1*/
+// ////function multiOverload(a: number): string;
+// /////** This is multiOverload F1 2*/
+// ////function multiOverload(b: string): string;
+// /////** This is multiOverload F1 3*/
+// ////function multiOverload(c: boolean): string;
+// /////** This is multiOverload Implementation */
+// ////function multiOverload(d): string {
+// ////    return "Hello";
+// ////}
+// ////multiOverl/*71*/oad(10);
+// ////multiOverl/*72*/oad("hello");
+// ////multiOverl/*73*/oad(true);
+// /////** This is ambient F1 1*/
+// ////declare function ambientF1(a: number): string;
+// /////** This is ambient F1 2*/
+// ////declare function ambientF1(b: string): string;
+// /////** This is ambient F1 3*/
+// ////declare function ambientF1(c: boolean): boolean;
+// /////*65*/
+// ////ambient/*74*/F1(10);
+// ////ambient/*75*/F1("hello");
+// ////ambient/*76*/F1(true);
+// ////function foo(a/*77*/a: i3) {
+// ////}
+// ////foo(null);
+
+// verify.quickInfos({
+//     1: ["function f1(a: number): number (+1 overload)", "this is signature 1"],
+//     2: "function f1(b: string): number (+1 overload)",
+//     3: ["function f1(a: number): number (+1 overload)", "this is signature 1"],
+//     "4q": "function f1(b: string): number (+1 overload)",
+//     o4q: ["function f1(a: number): number (+1 overload)", "this is signature 1"]
+// });
+
+// goTo.marker('4');
+// verify.currentSignatureHelpDocCommentIs("");
+// verify.currentParameterHelpArgumentDocCommentIs("");
+// goTo.marker('o4');
+// verify.currentSignatureHelpDocCommentIs("this is signature 1");
+// verify.currentParameterHelpArgumentDocCommentIs("param a");
+
+// verify.quickInfos({
+//     5: "function f2(a: number): number (+1 overload)",
+//     6: ["function f2(b: string): number (+1 overload)", "this is signature 2"],
+//     7: "function f2(a: number): number (+1 overload)",
+//     "8q": ["function f2(b: string): number (+1 overload)", "this is signature 2"],
+//     o8q: "function f2(a: number): number (+1 overload)"
+// });
+
+// goTo.marker('8');
+// verify.currentSignatureHelpDocCommentIs("this is signature 2");
+// verify.currentParameterHelpArgumentDocCommentIs("");
+
+// goTo.marker('o8');
+// verify.currentSignatureHelpDocCommentIs("");
+// verify.currentParameterHelpArgumentDocCommentIs("param a");
+
+// verify.quickInfos({
+//     9: "function f3(a: number): number (+1 overload)",
+//     10: "function f3(b: string): number (+1 overload)",
+//     11: "function f3(a: number): number (+1 overload)",
+//     "12q": "function f3(b: string): number (+1 overload)",
+//     o12q: "function f3(a: number): number (+1 overload)"
+// });
+
+// goTo.marker('12');
+// verify.currentSignatureHelpDocCommentIs
```

Comment:
Is this file meant to be here? The whole thing looks commented out (and possibly generated - but we don't want fourslash tests over 2400 lines long!)